cdap-proto/src/main/java/co/cask/cdap/proto/ProgramRunStatus.java

```diff
@@ -26,5 +28,26 @@
   SUSPENDED,
   COMPLETED,
   FAILED,
-  KILLED
+  KILLED;
+
+  /**
+   * Conversion from program run status to Workflow node status.
+   * @param status the program run status to be converted
+   * @return the converted Workflow node status
+   */
+  public static WorkflowNodeState.NodeStatus toNodeStatus(ProgramRunStatus status) {
+    switch(status) {
```

Comment:
Since we want the node status map to be returned from the `WorkflowContext`, we want this enum in cdap-api which cannot have classes from cdap-proto.